tests/integration/test_ingress_relation.py

```diff
@@ -25,22 +25,31 @@ async def test_ingress_relation(
         f"""\
     import pathlib
     import subprocess
+    import ops
     from any_charm_base import AnyCharmBase
     from ingress import IngressPerAppRequirer
     class AnyCharm(AnyCharmBase):
         def __init__(self, *args, **kwargs):
             super().__init__(*args, **kwargs)
             self.ingress = IngressPerAppRequirer(self, port=8080)
+            self.unit.status = ops.BlockedStatus("Waiting for ingress relation")
+            self.framework.observe(
+                self.on.ingress_relation_changed, self._on_ingress_relation_joined
```

Comment:
It's not really important, but it seems odd to be observing `ingress_relation_changed` but calling `_on_ingress_relation_joined`. Might be worth being consistent there.

Reply:
Ah my bad, was experimenting with different relation events and must have forgotten to rename the method